posthog.go

```diff
@@ -296,6 +299,27 @@ func (c *client) ReloadFeatureFlags() error {
 	return nil
 }
 
+func (c *client) GetFeatureFlagPayload(flagConfig FeatureFlagPayload) (interface{}, error) {
+	if err := flagConfig.validate(); err != nil {
+		return "", err
+	}
```

Comment:
I'd be inclined to return `false` instead of `""` here, since we're returning the more permissive `interface{}` type.  This is also the pattern we use elsewhere throughout this SDK, e.g. [here](https://github.com/PostHog/posthog-go/blob/master/posthog.go#L300-L302)